monthly-contributions/src/main/scala/com/gu/support/workers/lambdas/CreateZuoraSubscription.scala

```diff
@@ -27,7 +27,7 @@ class CreateZuoraSubscription(servicesProvider: ServiceProvider = ServiceProvide
     }
 
   def skipSubscribe(state: CreateZuoraSubscriptionState, subscription: SubscriptionResponse): Future[SendThankYouEmailState] = {
-    logger.info(s"Skipping subscribe for user because they are already a contributor " +
+    logger.info(s"Skipping subscribe for user ${state.user.id} because they are already an active contributor " +
```

Comment:
I think we want to avoid writing user ids into logs, just from a GDPR paranoia point of view.

Reply:
Sure, I'll remove this - although I do wonder how we are going to address this problem more generally (and also exactly what the status of account number will be in a GDPR compliant world!)